### DISCUSSION TITLE
Firefox freezes on any compilation error

### DISCUSSION THREAD

**Discussion by thany** [vercel/next.js#48660] [Help]
### Summary

So, Firefox 112 will happily soft-freeze by using every last bit of memory available, causing me to have to kill the process. 

This happens predictably when there is a compilation error. Something under water immediately starts hoggin one of my CPU cores, and all of my memory. I don't (yet) know what is causing this.

For me, this is 100% reproducible.

Unfortunately, it's pretty near impossible to come up with a repro scenario, as this didn't happen, of course, when the project was a bit smaller. Also doesn't seem to happen for the getting started guide, and other examples. I might help to know I'm using Turborepo, but the underlying stuff concerning Next.js is all bog standard.

So I guess I can't ask for a solution, given the structure of this project, given the fact that it seems impossible to create a repro repo, and given the fact that this repro is IP and not even on Github in the first place, so I can't really share it. 

I guess what I'm asking for, is directions on how to diagnose this. Or perhaps pointer on what *might* be causing this. Is there anything *at all*, no matter how benign, that could start hogging every last drop of resources when a compilation error occurs? Any little pointer might help.

### Additional information

```js
>npx next info

    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 18.15.0
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 13.2.4
      eslint-config-next: 13.1.6
      react: 18.2.0
      react-dom: 18.2.0
```
```


### Example

_No response_

**Comment by ShellWen** [comment]
I went to the same trouble, I can't do anything except close the tab
